Package@swift-5.6.swift

```diff
@@ -0,0 +1,51 @@
+// swift-tools-version: 5.6
```

Comment:
The `@swift-5.6` suffix is canonical?

Reply:
What do you mean with canonical? SPM used these suffixes to identify what manifest to pick for a given swift version

Reply:
ah, yea I did not realize that swiftly uses those suffixes, seems reasonable then